samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -57,14 +57,14 @@ See `Fast Pair Advertising`_ for detailed information about discoverable and not
 Fast Pair device registration
 =============================
 
-Before a device can be used as a Fast Pair Provider, you must register the device model with Google.
+Before you can use a device as a Fast Pair Provider, you must register the device model with Google.
 This is required to obtain Model ID and Anti-Spoofing Private Key.
 See :ref:`ug_bt_fast_pair_provisioning` in the Fast Pair user guide for details.
 
 .. tip::
    The sample provides TX power in the Bluetooth advertising data.
    There is no need to provide the TX power value during device model registration.
-   The device is using only Bluetooth LE, so you must select **Skip connecting audio profiles (e.g. A2DP, HFP)** option when registering the device.
+   The device is using only Bluetooth LE, so you must select **Skip connecting audio profiles (for example, A2DP or HFP)** option when registering the device.
```

Comment:
If this is how the option is named, we can't correct it in documentation.

Reply:
Oh, I did not realize this was an option available in Google. Fixed.